src/runtime_src/core/common/drv/fast_adapter.c

```diff
@@ -191,7 +191,7 @@ int xrt_cu_fa_init(struct xrt_cu *xcu)
 		goto err;
 	}
 
-	/* TODO: Write 0 to MSWR resiger is a hack.
+	/* TODO: Write 0 to MSWR resiger is a hack for unknown hardware issue.
```

Comment:
Is "resiger" is a hack for an unknown hardware device? :-)

Reply:
Thanks. Fixed